site/public/js/drag-and-drop.js

```diff
@@ -1051,3 +1051,68 @@ function handleUploadCourseMaterials(csrf_token, expand_zip, cmPath, requested_p
         }
     });
 }
+
+function handleUploadGradeablesZip() {
+  var submit_url = buildCourseUrl(['homework/library', 'zip', 'upload']);
```

Comment:
These urls may need to change idk. But that can be done when we tie everything together easy enough.